sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -762,8 +763,11 @@ ansiReserved
     | USING | WHEN | WHERE | WITH
     ;
 
+// When `spark.sql.parser.ansi.enabled` = true, the `ansiNonReserved` keywords can be used for identifiers.
```

Comment:
so `ansiReserved` + `ansiNonReserved` are all the keywords in Spark? Can we add a note here? If in the future someone adds a new keyword, he must put it either in `ansiReserved` or in `ansiNonReserved`.

Reply:
> so `ansiReserved` + `ansiNonReserved` are all the keywords in Spark?

Yeah, yes.

> Can we add a note here? If in the future someone adds a new keyword, he must put it either in `ansiReserved` or in `ansiNonReserved`.

ok.